### PR TITLE
fix .children_with_depth() method

### LINKED ISSUE #3: `.children()` is off
The following isn't working right now:
```rust
flat_tree::children(9) == (8, 10)
```
Instead it's returning `(2, 4)`

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Fixes #3. Thanks!

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
`v2.0.0`

File: src/lib.rs [hunk 1]
```diff
@@ -127,8 +127,8 @@ pub fn children_with_depth(i: u64, depth: u64) -> Option<(u64, u64)> {
   } else if depth == 0 {
     Some((i, i))
   } else {
-    let off = offset_with_depth(i, depth) >> 1;
-    Some((index(depth - 1, off), index(depth - 1, off + 1)))
+    let offset = offset_with_depth(i, depth) * 2;
```
Comment (mafintosh) [inline src/lib.rs]: `<< 2` would also do the job :)

Comment (yoshuawuyts) [reply src/lib.rs]: Oh sick! Yeah, I gotta brush up on my binary operators lolol.

Comment (yoshuawuyts) [reply src/lib.rs]: Oh no, that outputs `(16, 18)` instead of `(8, 10)`. Not sure it's quite the same then :/

**Pull request merged**